apiserver/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM scratch as source
+
+ARG BIN_DIR
+ARG TARGETARCH
+
+COPY ${BIN_DIR}/apiserver-${TARGETARCH} /code/apiserver
+COPY ${BIN_DIR}/filecheck-${TARGETARCH} /code/filecheck
+
+FROM calico/base
+
+COPY --from=source / /
+
```

Can we add these container labels please?  This will bring this image into line with our other images and is required for Openshift certification.

```suggestion
ARG GIT_VERSION=unknown

LABEL description="Aggregated API server which enables calico resources to be managed via kubectl"
LABEL maintainer="maintainers@tigera.io"
LABEL name="Calico APIserver"
LABEL release="1"
LABEL summary="Calico APIserver"
LABEL vendor="Project Calico"
LABEL version=${GIT_VERSION}
```

---

Labels are added. Thanks for your suggestion.